File: JavaWeb2-eureka-client-user/src/main/java/com/javaweb/web/eo/operationLog/OperationLogListRequest.java
package com.javaweb.web.eo.operationLog;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OperationLogListRequest implements Serializable {

	private static final long serialVersionUID = -1259214841753949424L;
	
	private String userName;//用户名
	
	private String requestIpAddress;//请求IP地址
	
	private Long currentPage = 1L;//默认当前第1页
	
	private Long pageSize = 10L;//默认每页显示10条
	
}

File: JavaWeb2-angular/src/app/model/operationLog/OperationLogListRequest.ts
export class OperationLogListRequest {

  userName:string;//用户名

  requestIpAddress:string;//请求IP地址

  createStartDate:any = null;//操作日志创建的开始日期

  createEndDate:any = null;//操作日志创建的结束日期

  currentPage = 1;//当前页数

  pageSize = 10;//每页显示多少条

}
